CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * [Added] The ability to customize the disabled colors of the primary button with `PaymentSheetAppearance.primaryButton.disabledBackgroundColor` and `PaymentSheetAppearance.primaryButton.disabledTextColor`.
 * [Added] CVC Recollection is now in GA. For more information see our docs for [here](https://docs.stripe.com/payments/accept-a-payment?platform=ios#ios-cvc-recollection) for intent first integrations or [here](https://docs.stripe.com/payments/accept-a-payment-deferred?platform=ios&type=payment#ios-cvc-recollection) for deferred intent integrations.
 * [Fixed] Fixed an issue where checkboxes were not visible when `appearance.colors.componentBorder` was transparent.
+* [Fixed] Fixes an issue where ISK was not correctly formatted as a zero-decimal currency when using PaymentSheet or Apple Pay. (Thanks [@doganaltinbas](https://github.com/doganaltinbas)!)
```

Comment:
Are the two of us who have fixed this issue, or is it just a wrong copy/past? 🤔

Reply:
Sorry, copied out of the wrong tab! Thanks for catching that! 😓

Reply:
Thank you 😊